.github/workflows/build.yaml

```diff
@@ -1,23 +1,63 @@
-name: Build Content
+name: Gating
 on:
   push:
-    branches: [ master ]
+    branches: [ '*' ]
   pull_request:
     branches: [ master ]
 jobs:
-  validate:
-    name: Build Content
+  validate-ubuntu:
+    name: Build, Test, Lint Ansible Roles on Ubuntu 20.04
     runs-on: ubuntu-20.04
     steps:
       - name: Install Deps
         uses: mstksg/get-package@master
         with:
-                apt-get: cmake ninja-build libopenscap8 libxml2-utils expat xsltproc python3-jinja2 python3-yaml ansible-lint python3-github
+          apt-get: cmake ninja-build libopenscap8 libxml2-utils expat xsltproc python3-jinja2 python3-yaml ansible-lint python3-github
       - name: Checkout
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
       - name: Build
-        run: |
-          ./build_product rhel7 rhel8
+        run: ./build_product chromium fedora firefox fuse6 rhcos4 rhel7 rhel8 rhel9 rhosp10 rhosp13 sle12 sle15 ubuntu2004
       - name: Test
         run: ctest -j2 --output-on-failure -E unique-stigids
         working-directory: ./build
+      - name: Build Ansible Roles
+        run: PYTHONPATH=. python3 utils/ansible_playbook_to_role.py --build-playbooks-dir ./build/ansible/ --dry-run ./build/ansible_roles
+      - name: Lint Ansible Roles
+        run: ansible-lint -x 204 -x experimental ./build/ansible_roles/*
```

Comment:
@evgenyz We discussed this today, and I think we decided to have it as nightly job for now, right? Or it will remain here until we have something ran nightly on Jenkins?
So could you move the Ansible Roles Lint test out of the gating?

Reply:
Yeah, yeah, forgot to remove it from PR. Done.

Reply:
Is it a good idea to remove `.github/workflows/validate_ansible_roles.yaml` while there is not yet a replacement for it?

Reply:
I'm fine removing it if you plan to add it to Jenkins soon-ish.

Reply:
Err, it's already in the history of the code. For the lifetime of the repo :)

Reply:
And we don't want it to be in Actions anyway

Reply:
I'm planing to add it to Go-CD